docs/source/en/using-diffusers/workflows.md

```diff
@@ -0,0 +1,331 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Shareable workflows
+
+<Tip warning={true}>
+
+🧪 Workflow is experimental and its APIs can change in the future.
+
+</Tip>
+
+Workflows provide a simple mechanism to share your pipeline call arguments and scheduler configuration, making it easier to reproduce results. 
+
+## Serializing a workflow
+
+```python
+from diffusers import DiffusionPipeline
+import torch
+
+pipeline = DiffusionPipeline.from_pretrained(
+    "runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16, safety_checker=None
+).to("cuda")
+
+outputs = pipeline("A painting of a horse", num_inference_steps=15, return_workflow=True)
+workflow = outputs.workflow
+```
+
+If you look at this specific workflow, you'll see values like the number of inference steps, guidance scale, and height and width as well as the scheduler details:
+
+```bash
+{'prompt': 'A painting of a horse',
+ 'height': None,
+ 'width': None,
+ 'num_inference_steps': 15,
+ 'guidance_scale': 7.5,
+ 'negative_prompt': None,
+ 'eta': 0.0,
+ 'latents': None,
+ 'prompt_embeds': None,
+ 'negative_prompt_embeds': None,
+ 'output_type': 'pil',
+ 'return_dict': True,
+ 'callback': None,
+ 'callback_steps': 1,
+ 'cross_attention_kwargs': None,
+ 'guidance_rescale': 0.0,
+ 'clip_skip': None,
+ 'generator_seed': 331018828,
+ 'generator_device': device(type='cpu'),
+ '_name_or_path': 'runwayml/stable-diffusion-v1-5',
+ 'scheduler_config': FrozenDict([('num_train_timesteps', 1000),
+             ('beta_start', 0.00085),
+             ('beta_end', 0.012),
+             ('beta_schedule', 'scaled_linear'),
+             ('trained_betas', None),
+             ('skip_prk_steps', True),
+             ('set_alpha_to_one', False),
+             ('prediction_type', 'epsilon'),
+             ('timestep_spacing', 'leading'),
+             ('steps_offset', 1),
+             ('_use_default_values', ['prediction_type', 'timestep_spacing']),
+             ('_class_name', 'PNDMScheduler'),
+             ('_diffusers_version', '0.6.0'),
+             ('clip_sample', False)])}
+```
+
+A [`Workflow`] object provides all the argument values in the `__call__()` of a pipeline. Add `return_workflow=True` to return a `Workflow` object:
```

Comment:
I think this sentence makes more sense at the beginning of the "Serializing a workflow" section as an introduction instead of just starting with a code snippet with no context.